src/pheval/analyse/analysis.py

```diff
@@ -0,0 +1,779 @@
+# #!/usr/bin/python
+import csv
+import itertools
+from collections import defaultdict
+from dataclasses import dataclass, field
+from pathlib import Path
+from statistics import mean
+
+import click
+import pandas as pd
+
+from ..utils.file_utils import all_files, files_with_suffix, obtain_closest_file_name
```

Comment:
Maybe we can consider importing functions using the package name instead the relative path.

Reply:
Good idea!